hugo/content/docs/learn/putting-it-together.md

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Putting It Together'
+---
+
+**The moment of truth has arrived!** In this final section, we'll connect all the pieces we've built and bring our interactive diagram to life.
+
+![Final Diagram](/images/final-diagram.png)
```

Comment:
should this be the same as on the index page?

Reply:
Yes, it should be the same